components/Report/interfaces.rst

```diff
@@ -0,0 +1,68 @@
+Interfaces
+==========
+
+Model Interfaces
+----------------
+
+.. _component_report_model_report-interface:
+
+ReportInterface
+~~~~~~~~~~~~~~~
+
+This interface should be implemented by model representing a single Report.
+
+.. note::
+    For more detailed information go to `Sylius API ReportInterface`_.
+
+.. _Sylius API ReportInterface: http://api.sylius.org/Sylius/Component/Report/Model/ReportInterface.html
+
+Service Interfaces
+------------------
+
+.. _component_report_data-fetcher_data-fetcher-interface:
+
+DataFetcherInterface
+~~~~~~~~~~~~~~~~~~~~
+
+This interface should be implemented by service representing data fetcher.
+
+.. note::
+    For more detailed information go to `Sylius API DataFetcherInterface`_.
+
+.. _Sylius API DataFetcherInterface: http://api.sylius.org/Sylius/Component/Report/DataFetcher/DataFetcherInterface.html
+
+.. _component_report_renderer_renderer-interface:
+
+RendererInterface
+~~~~~~~~~~~~~~~~~
+
+This interface should be implemented by service representing data renderer.
```

Comment:
report renderer*

, which renders the data in a specific output format. Examples would be "graph", "table", "csv".